.vib/phpbb/cypress/cypress.json

```diff
@@ -0,0 +1,8 @@
+{
+  "baseUrl": "http://localhost",
+  "defaultCommandTimeout": 30000,
```

Comment:
Which is the aim of it?

Reply:
The aim is to reduce flakiness, we usually add this to other assets too.